must-gather/collection-scripts/gather

```diff
@@ -56,7 +60,7 @@ cluster_resources+=(ns/$CMA_NS)
 log "Collecting information for namespace ${CMA_NS}" >> "${BASE_COLLECTION_PATH}/gather-debug.log"
 for resource in ${cluster_resources[@]}; do
   log "Inspecting $resource" >> "${BASE_COLLECTION_PATH}/gather-debug.log"
-  oc adm inspect --dest-dir ${BASE_COLLECTION_PATH} -n ${CMA_NS} ${resource} >> "${BASE_COLLECTION_PATH}/gather-debug.log" 2>&1
+  oc adm inspect --cache-dir=${KUBECACHEDIR} --dest-dir ${BASE_COLLECTION_PATH} -n ${CMA_NS} ${resource} >> "${BASE_COLLECTION_PATH}/gather-debug.log" 2>&1
```

Comment:
I just noticed this... Does the `-n ${CMA_NS}` mean that it will only get resources from that namespace? We want the cluster resources from all namespaces.

Reply:
Correct. Addressed in https://github.com/openshift/custom-metrics-autoscaler-operator/pull/23/commits/852fce5e9a0239715093e48e112be69fe54c7e91

Reply:
For my education; Could the CR (kedacontroller, scaledjob, scaledobject, etc.) exist in a namespace different from the one where the subscription was installed?

Reply:
Yes. Only the kedacontroller is expected in the NS with the subscription. The rest of the objects are expected to live in the NS where the managed workloads are running. The controller watches objects across all namespaces.